packages/typespec-vscode/src/code-action-provider.ts

```diff
@@ -46,10 +46,30 @@ export class TypeSpecCodeActionProvider implements vscode.CodeActionProvider {
           ),
         );
       }
+      if (
+        diagnostic.source === "TypeSpec" &&
+        diagnostic.code &&
+        diagnostic.code === "import-not-found"
```

Comment:
user may import files instead of package. we should only provide this quick fix when user import packages